plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -73,3 +73,64 @@ Compressor.Threshold="Threshold (dB)"
 Compressor.AttackTime="Attack (ms)"
 Compressor.ReleaseTime="Release (ms)"
 Compressor.OutputGain="Output Gain (dB)"
+Crosshatching="Crosshatching"
+Crosshatching.Offset="Offset"
+Crosshatching.Threshold1="Threshold 1"
+Crosshatching.Threshold2="Threshold 2"
+Crosshatching.Threshold3="Threshold 3"
+Crosshatching.Threshold4="Threshold 4"
+CrossStitching="Cross Stitching"
+CrossStitching.StitchGap="Stitch Gap"
+CrossStitching.NegativeBias="Negative Bias"
+CrossStitching.StitchColor="Stitch Color"
+CrossStitching.StitchOpacity="Stitch Opacity"
+CrossStitching.InvertStitching="Invert Pattern"
+CrossStitching.Grayscale="Grayscale"
+DrosteEffect="Droste Effect"
+DrosteEffect.NumberOfRepeats="No. of Repeats"
+DrosteEffect.DownscalePer="Downscale %"
+DrosteEffect.XOffset="X-Offset %"
```

Comment:
Consistency nitpick.  Here you use "X-Offset" and "Y-Offset" while further down you use "X Offset" and "Y Offset".  I have no preference on either, but it should be consistent.